.github/workflows/windows-rubygems.yml

```diff
@@ -41,6 +41,10 @@ jobs:
         with:
           toolchain: ${{ matrix.cargo.toolchain }}-${{ matrix.cargo.target }}
           default: true
+      - name: Install deps for Rust
+        if: ${{ matrix.ruby.name != "3.1" }}
+        run: |
+          pacman -S mingw-w64-x86_64-clang
```

Comment:
It's necessary to remove the `mingw-w64-x86_64-clang` package and then install the `ucrt` one like here: https://github.com/larskanis/rubygems/runs/4985946989?check_suite_focus=true#step:5:2

So this should do the trick:
```
  pacman --remove --cascade mingw-w64-x86_64-clang
  pacman --sync --noconfirm --needed $ENV:MINGW_PACKAGE_PREFIX-clang
```

The necessity to remove the non-ucrt package is something that should be fixed (in clang-sys if I understand the code right). It's possible to have both ucrt and non-ucrt packages installed and one shouldn't disturb the other one.

Reply:
Wow thank you for the quick, and thorough response! Will give this a shot and report back.